README.md

```diff
@@ -336,13 +307,13 @@ When running in Openshift, we need to grant the appropriate security context for
    oc adm policy add-scc-to-user privileged system:serviceaccount:submariner:submariner-engine 
    ```
    
-# Building/Contributing
+# Building
 
 To build `submariner-engine` and `submariner-route-agent` you can trigger `make`, which will perform a Dapperized build of the components.
```

Comment:
I'd talk here instead about how to build the operator.

Reply:
I think all operator related content should be addressed in a separate PR.

Reply:
Since this is a small part, I did it here.